### PR TITLE
Delete supplier

### PR DESCRIPTION
**Pull request by alexandrelissowski**
We introduce the capability to delete a supplier. At this occasion I modify the former article _create supplier_ onto _manage supplier_ and I add some screenshots.

### CODE REVIEW

**Comment by alexandrelissowski** [comment]
English request

- Name: Alexandre
- Wordcount: 600 w
- Deadline: 30 October
- T-shirt size: S

**Comment by nestorAkeneo** [comment]
Your proofreading request has been created

Ticket link: https://akeneo.atlassian.net/browse/ENG-132:

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## Use flat-file import
+
+Otherwise, you can use `CSV` or `XLSX` files to import suppliers.
+You can download a template containing the header with the code for each column.
+
+When you fill in the columns _supplier_contact_emails_, _retailer_contact_usernames_, _catalog_locales_, _families_, the values must be separated by comas.
+
+And the column _allow_product_creation_ is the boolean with 0 for _No_ and 1 for _Yes_.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:63]:
```suggestion
And the _allow_product_creation_ column is boolean with 0 for _No_ and 1 for _Yes_.
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+![Create a supplier with the User Interface](../img/RETAILER_Suppliers_Suppliers_CreateASupplier.png)
+
+
+## Use flat-file import
+
+Otherwise, you can use `CSV` or `XLSX` files to import suppliers.
+You can download a template containing the header with the code for each column.
+
+When you fill in the columns _supplier_contact_emails_, _retailer_contact_usernames_, _catalog_locales_, _families_, the values must be separated by comas.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:61]:
```suggestion
When you fill in the _supplier_contact_emails_, _retailer_contact_usernames_, _catalog_locales_, _families_, columns the values must be separated by comas.
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+And the column _allow_product_creation_ is the boolean with 0 for _No_ and 1 for _Yes_.
+
+::: warning
+Formerly the column _supplier_contact_emails_ was named _supplier_referent_email_. To continue using the same import file you were using, you must change this column header.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:66]:
```suggestion
Formerly the _supplier_contact_emails_ column was named _supplier_referent_email_. To continue using the same import file you were using, you must change this column header.
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+:::
+
+Then you can create your suppliers with the file you completed by following these steps:
+1. Access to your Onboarder from the PIM,
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:70]:
```suggestion
1. Access to your Onboarder from the PIM,
```
```suggestion
1. Access your Onboarder from the PIM,
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+1. Click on `Upload a file`,
+1. Drag your file in the drop area or browse disk,
+1. Click on `Upload and import now`.
+
+You are redirected to the Process tracker page, where you can follow the import progress and your suppliers are created.
+
+
+# Delete supplier
+During your use of Onboarder, you may need to delete suppliers for different reasons. There are two ways to proceed:
+- From the supplier page
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:82]: Sometimes you say 'Suppliers page' (l.71) sometimes, 'supplier page' -> I don't know which is right :)

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+![Delete from supplier page](../img/RETAILER_DeleteSupplier_Form.png)
+- From suppliers list
+![Delete from supplier list](../img/RETAILER_DeleteSupplier_List.png)
+
+In both cases, you'll need to confirm this action.
+![Modal confirmation](../img/RETAILER_DeleteSupplier_Modal.png)
+
+Once you choose to delete a supplier, there are several consequences you should be aware of:
+1. All the supplier users connected to this supplier will be unlinked from it and no longer have access to Akeneo Onboarder,
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:91]:
```suggestion
1. All the supplier users connected to this supplier will be unlinked from it and will no longer have access to Akeneo Onboarder,
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+1. The _supplier_ and _supplier ref_ of products that are assigned to the deleted supplier will be purged in the same time,
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:92]:
```suggestion
1. The _supplier_ and _supplier ref_ of products that are assigned to the deleted supplier will be purged at the same time,
```

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+1. All data stored in Akeneo Onboarder will be deleted (products, their values and assets). Make sure they completed all required data, and you have it into the PIM before you proceed to deletion.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:93]: unclear (2nd sentence)

File: content/md/retailer-onboarder/suppliers-management/manage-supplier.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+::: warning
+If, after some time, you decide to create again a supplier that has been previously removed, the data won't be recovered as a new supplier will be created from scratch.
```
Comment (camillebourjade) [inline content/md/retailer-onboarder/suppliers-management/manage-supplier.md:96]:
```suggestion
If, after some time, you decide to create a supplier that has been previously removed, the data won't be recovered as a new supplier will be created from scratch.
```

**Review by camillebourjade — APPROVED** [review]
English review

**Pull request merged**